src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -179,9 +181,10 @@ public function runPending(array $migrations, array $options = [])
      * @param  string  $file
      * @param  int  $batch
      * @param  bool  $pretend
+     * @param  bool  $fake
      * @return void
      */
-    protected function runUp($file, $batch, $pretend)
+    protected function runUp($file, $batch, $pretend, $fake)
```

Comment:
This is a breaking change unfortunately.

Reply:
I guess because this method is being called from somewhere else?  I was wondering about that and searched for any other calls to this method, but didn't find anything.  Thanks for letting me know.  Can we set a default value to make the argument optional?

```suggestion
    protected function runUp($file, $batch, $pretend, $fake = false)
```

Reply:
Changing a method signature is a breaking change for anyone overriding it.

Reply:
Ahh, got it, thanks.  I just pushed an update that creates new fakeRunUp and fakeRunDown methods instead of changing the existing method signatures.